data-prepper-api/build.gradle

```diff
@@ -14,6 +14,8 @@ dependencies {
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
     implementation libs.parquet.common
     implementation libs.commons.lang3
+    implementation 'jakarta.validation:jakarta.validation-api:3.0.2'
+    implementation 'org.hibernate.validator:hibernate-validator:8.0.1.Final'
```

Comment:
I don't think you need the Hibernate dependency here. We only need this when evaluating.